packages/smooth_app/lib/pages/product/common/product_list_page.dart

```diff
@@ -53,6 +53,19 @@ class _ProductListPageState extends State<ProductListPage>
     productList = widget.productList;
   }
 
+  //returns bool to handle WillPopScope
+  Future<bool> _handleUserBacktap() async {
```

Comment:
Please use documentation comments to document functions and classes (`///` instead of `//`)